src/main/java/io/antmedia/datastore/db/DataStore.java

```diff
@@ -1238,29 +1256,93 @@ else if (sortBy.equals("endDate")) {
 	/**
 	 * Creates new P2PConnection
 	 * @param conn - P2PConnection object
-	 * @return boolean - success 
+	 * @return boolean - success
 	 */
 	public abstract boolean createP2PConnection(P2PConnection conn);
 
 	/**
 	 * Get the P2PConnection by streamId
 	 * @param streamId - stream id for P2PConnection
-	 * @return P2PConnection - if exist else null 
+	 * @return P2PConnection - if exist else null
 	 */
 	public abstract P2PConnection getP2PConnection(String streamId);
 
 	/**
 	 * Deletes a P2PConnection
 	 * @param conn - P2PConnection object
-	 * @return boolean - success 
+	 * @return boolean - success
 	 */
 	public abstract boolean deleteP2PConnection(String streamId);
 
+	/**
+	 * Add a stream into publisher request list
+	 * @param mainTrackId - main track id
+	 * @param streamId - stream id
+	 * @return boolean - success
+	 */
+	public abstract boolean addIntoPublisherRequestList(String mainTrackId, String streamId);
```

Comment:
I think we don't need to additional methods for the new fields, we can just use the updateBroadcast method